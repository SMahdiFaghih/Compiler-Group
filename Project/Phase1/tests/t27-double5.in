147852e+10