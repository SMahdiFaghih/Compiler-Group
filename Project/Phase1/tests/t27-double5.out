T_INTLITERAL 147852
T_ID e
+
T_INTLITERAL 10
